mabar-frontend/src/services/aiMatchmakingService.ts

```
@@ -0,0 +1,456 @@
+   * Execute the toolbox action based on AI request
+   */
+  private static async executeToolboxAction(aiRequest: AIRequest): Promise<AIResponse> {
+    const { action, parameters } = aiRequest
+
+    try {
+      let response: AIResponse
+      
+      switch (action) {
+        case 'getAvailableVenues':
+          const venuesResult = await MatchmakingToolboxService.getAvailableVenues(parameters)
+          response = { text: '', data: venuesResult.data, isEmpty: venuesResult.isEmpty }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)